views/home.php

```diff
@@ -204,14 +205,16 @@ function showSelectedCourses(){
         };
 
         $(document).ready(function(){
-            var courses = localStorage.getItem("courses");
-            if (courses){
-                var courses = courses.split(",");
-                courses.forEach(function(course, index){
-                    toggleCourseCSV(course);
-                    $("#"+course).addClass("active");
-                });
-            }
+        	if (typeof(Storage) !== "undefined") {

```

Comment:
why arent you checking typeof(localStorage)?

Reply:
W3 reference uses this notation?